exercises/00-init/src/modules/users/users-actions.js

```diff
@@ -1,17 +1,14 @@
+import { createActions } from 'reduxsauce'
 
 
-const ADD_USER = 'ADD_USER'
-
-const addUser = (firstName, lastName) => ({
-    type: ADD_USER,
-    payload: {firstName, lastName}
-})
-
-export default {
-    Types: {
-      ADD_USER
+const { Types, Creators } = createActions(
+    {
+        addUser: ['firstName', 'lastName'],
     },
-    Creators: {
-        addUser
+    {   
+        prefix: "users",
+        namespace: "/"

```

Comment:
Does this actually do anything? I'm pretty sure reduxsauce does not have such an option, where did you find it?

Reply:
It doesn't do anyting :). I noticed, but then forgot to look into it.
I got it from https://redux-actions.js.org/api/createaction which I realize now that is not reduxsauce. 
So let's pretend this was never here :)